notNeededPackages.json

```diff
@@ -1076,6 +1076,10 @@
             "libraryName": "codegen.macro",
             "asOfVersion": "4.1.0"
         },
+        "colorbrewer": {
+            "libraryName": "colorbrewer",
+            "asOfVersion": "1.5.1"
```

Comment:
I don't see types in the published package at this version. They seem to only appear at 1.6.

Reply:
Updated.